products/statement-generator/src/components-layout/Affirmation.tsx

```diff
@@ -59,10 +59,11 @@ const Affirmation = () => {
         <p>{t(affirmationData.description)}</p>
       </div>
 
-      <div className={utilityClasses.buttonContainer}>
+      <div
+        className={`${utilityClasses.buttonContainer} ${utilityClasses.btnJustifyRight}`}
```

Comment:
You have different className than the style you implemented

Reply:
I fixed this issue, thanks for catching it! I'm a little confused on your request to clean up the commits.  I'd love to do it, may I have a little more clarity on this?